projects/planner/src/app/shared/learn/quiz.service.ts

```diff
@@ -55,29 +56,18 @@ export class QuizItem {
         // Equivalent narcotic depth:  this.gasProperties.end
         // Maximum narcotic depth this.gasProperties.mnd
 
+        const tools: QuizItemTools = {
+            depthConverter: this.depthConverter,
+            nitroxCalculator: this.nitroxCalculator,
+            sacCalculator: this.sacCalculator,
+            gasProperties: this.gasProperties
+        };
 
-        if (this.categoryName.toLowerCase().includes('maximum operational depth')) {
-            return this.nitroxCalculator.mod(this.variables[0], this.variables[1]);
+        if (typeof this.template.calculateAnswer === 'function') {
```

Comment:
Not needed check, if the calculateAnwer is not a function, than whole definition is wrong, there will be test for that.